src/tactic/linear_combination.lean

```diff
@@ -354,6 +368,9 @@ setup_tactic_parser
   prove their target using the linear combination instead of attempting to
   finish the proof.
 
+Users may provide an optional `with exponent n`. This will raise the goal to the power `n`
+  before subtracting the linear combination.
```

Comment:
Does it do this before or after rearranging to the form `_ = 0`? I can't see any tests where the goal isn't already in this form.

Reply:
It does it after. The second test `example (x y z : ℚ) (h : x = y) (h2 : x * y = 0) : y*z = -x :=` tests this, although it's obscured by testing the config argument parsing at the same time.